gherkin/php/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "cucumber/gherkin",
+    "description": "Gherkin parser",
+    "author": "Cucumber Limited <cukes@googlegroups.com>",
+    "license": "MIT",
+    "type": "library",
+    "autoload": {
+        "psr-4": {
+            "Cucumber\\Gherkin\\": [
+                "src/",
+                "src-generated/"
+            ]
+        }
+    },
+    "require": {
+        "php": "^8.1",
+        "ext-mbstring": "*",
+        "cucumber/messages": "dev-main||^17.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "vimeo/psalm": "^4.20",
+        "friendsofphp/php-cs-fixer": "^3.5",
+        "psalm/plugin-phpunit": "^0.16.1"
+    },
+    "repositories": [
+        {
+            "type": "path",
+            "url": "../../messages/php"
+        }
+    ]
```

Comment:
What is the meaning of this?

Reply:
This means when working on `cucumber/common` and typing `composer install`, it will symlink the messages into the `vendor` folder rather than downloading the package.

This is only when working on a checkout of `common`, not if requiring `cucumber/gherkin` as a package in some other project